### PR TITLE
[fix] ynh_local_curl

### PR DESCRIPTION
**Pull request by kay0u**
## The problem

In some conditions, ynh_local_curl doesn't curl the right url

## Solution

I'm ensure that the url is well formatted

## PR Status

Tested

## How to test

```
path_url=/test
ynh_local_curl index.php
```

```
path_url=/
ynh_local_curl index.php
```

```
path_url=/
ynh_local_curl /index.php
```

## Validation

- [ ] Principle agreement 0/2 : 
- [ ] Quick review 0/1 : 
- [ ] Simple test 0/1 : 
- [ ] Deep review 0/1 :

### CODE REVIEW

**Comment by alexAubin** [comment]
Anyone from @YunoHost/apps can review this ?

File: data/helpers.d/utils [hunk 1]
```diff
@@ -257,7 +257,15 @@ ynh_setup_source () {
 # | arg: ...         - (Optionnal) More POST keys and values
 ynh_local_curl () {
 	# Define url of page to curl
-	local full_page_url=https://localhost$path_url$1
+	path_url=$(ynh_normalize_url_path $path_url)
```
Comment (maniackcrudelis) [inline data/helpers.d/utils]: path_url shouldn't be modified here, it should already have been normalized earlier in the install script.

**Comment by maniackcrudelis** [comment]
Could you give an example of curl fetching the wrong url ?
All I can see in your code is to prevent a double slash after the path_url. Not sure though it's really a big deal.

File: data/helpers.d/utils (hunk 1 above)
Comment (kay0u) [reply data/helpers.d/utils]: Ok, I'll remove this line

**Comment by kay0u** [comment]
> Could you give an example of curl fetching the wrong url ?
> All I can see in your code is to prevent a double slash after the path_url. Not sure though it's really a big deal.

[dotclear](https://github.com/YunoHost-Apps/dotclear2_ynh/blob/testing/scripts/install#L149)

This app doesn't well configured with a double slash (Don't know why)

**Comment by maniackcrudelis** [comment]
LGTM, not tested though

**Comment by maniackcrudelis** [comment]
Bump

@YunoHost/apps

**Comment by alexAubin** [comment]
#Yolo

**Pull request merged**